packages/node_modules/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -0,0 +1,322 @@
+          highlight_id: voiceaPayload.highlight.highlight_id,
+          text: voiceaPayload.highlight.transcript,
+          highlightLabel: voiceaPayload.highlight.highlight_label,
+          highlightSource: voiceaPayload.highlight.highlight_source
+        });
+        break;
+
+      case TRANSCRIPTION_TYPE.EVA_THANKS:
+        Trigger.trigger(this, {
+          file: 'voicea',
+          function: 'processTranscription'
+        },
+        EVENT_TRIGGERS.EVA_COMMAND, {
+          isListening: false, text: voiceaPayload.command_response
+        });
+        break;
+
+      case TRANSCRIPTION_TYPE.EVA_WAKE:
+      case TRANSCRIPTION_TYPE.EVA_CANCEL:
+        Trigger.trigger(this, {
+          file: 'voicea',
+          function: 'processTranscription'
+        },
+        EVENT_TRIGGERS.EVA_COMMAND, {
+          isListening: voiceaPayload.type === TRANSCRIPTION_TYPE.EVA_WAKE
+        });
+        break;
+
+      default:
+        break;
+    }
+  };
+
+  /**
+   * Processes Caption Language Response
+   * @param {CaptionLanguageResponse} voiceaPayload
+   * @returns {void}
+   */
+  private processCaptionLanguageResponse = (voiceaPayload: CaptionLanguageResponse):void => {
+    if (voiceaPayload.statusCode === 200) {
+      Trigger.trigger(this, {
+        file: 'voicea',
+        function: 'processCaptionLanguageResponse'
+      },
+      EVENT_TRIGGERS.CAPTION_LANGUAGE_UPDATE, {statusCode: 200});
+    }
+    else {
+      Trigger.trigger(this, {
+        file: 'voicea',
+        function: 'processCaptionLanguageResponse'
+      },
+      EVENT_TRIGGERS.CAPTION_LANGUAGE_UPDATE, {statusCode: voiceaPayload.errorCode, errorMessage: voiceaPayload.message});
+    }
+  };
+
+  /**
+ * processes voicea announcement response and triggers event
+ * @param {Object} voiceaPayload
+ * @returns {void}
+ */
+  private processAnnouncementMessage = (voiceaPayload: AnnouncementPayload):void => {
+    const voiceaLanguageOptions = {
+      captionLanguages: voiceaPayload?.translation?.allowed_languages ?? [],
+      maxLanguages: voiceaPayload?.translation?.max_languages ?? 0,
+      spokenLanguages: voiceaPayload?.ASR?.spoken_languages ?? [],
+    };
+
+    Trigger.trigger(
+      this,
+      {
+        file: 'voicea',
+        function: 'processAnnouncementMessage'
+      },
+      EVENT_TRIGGERS.VOICEA_ANNOUNCEMENT,
+      voiceaLanguageOptions
+    );
+  };
+
+
+  /**
+   * Sends Announcement to add voicea to the meeting
+   * @returns {void}
+   */
+  private sendAnnouncement = ():void => {
+    if (this.hasVoiceaJoined) return;
+    this.socket.send({
+      id: `${this.seqNum}`,
+      type: 'publishRequest',
+      recipients: {
+        route: this.getBinding(),
+      },
+      headers: {},
+      data: {
+        clientPayload: {
+          version: 'v2',
+        },
+        eventType: 'relay.event',
+        relayType: VOICEA_RELAY_TYPES.CLIENT_ANNOUNCEMENT,
+      },
+      trackingId: `WEBCLIENT_${uuid.v4().toString()}`,
```

Comment:
issue: The trackingIdPrefix is set by the config of the SDK.  You should use this rather than "WEBCLIENT"

Reply:
removed

Reply:
This is still not correct. The tracking id prefix is part of the config `webex.config.trackingIdPrefix`